public/js/pimcore/object/tags/advancedManyToManyRelation.js

```diff
@@ -117,6 +117,7 @@ pimcore.object.tags.advancedManyToManyRelation = Class.create(pimcore.object.tag
 
         var visibleFieldsCount = columns.length;
 
+        var filterType = 'list';
```

Comment:
```suggestion
        let filterType = 'list';
```

Reply:
Would then be consistent with https://github.com/pimcore/admin-ui-classic-bundle/pull/659/files#diff-bea40e721430c1920c766df73237226f86c16a40e65012c1051c2cdff1ee8eaeR160 too